ts/components/SendSupportRequestOptions.tsx

```diff
@@ -7,30 +7,107 @@ import themeVariables from "../theme/variables";
 import ButtonDefaultOpacity from "./ButtonDefaultOpacity";
 import { H1 } from "./core/typography/H1";
 import { BaseHeader } from "./screens/BaseHeader";
-import Accordion from "./ui/Accordion";
 import Markdown from "./ui/Markdown";
 import IconFont from "./ui/IconFont";
 import FooterWithButtons from "./ui/FooterWithButtons";
-import { Label } from "./core/typography/Label";
-import { RawCheckBox } from "./core/selection/RawCheckBox";
 import { EdgeBorderComponent } from "./screens/EdgeBorderComponent";
 import { IOStyles } from "./core/variables/IOStyles";
+import { RawCheckBox } from "./core/selection/RawCheckBox";
+import { Label } from "./core/typography/Label";
+import Accordion from "./ui/Accordion";
+
+const checkboxStyle = StyleSheet.create({
+  container: {
+    flexDirection: "row",
+    alignItems: "flex-start",
+    justifyContent: "space-evenly"
+  }
+});
+
+/**
+ * Checkboxes we need as of now
+ */
+
+export enum CheckboxIDs {
+  sendScreenshot = "sendScreenshot",
+  sendPersonalInfo = "sendPersonalInfo"
+}
+
+/**
+ * Checkbox+Label+Optional Accordion component
+ */
+
+type ContextualHelpCheckboxFormEntryProps = {
+  target: CheckboxIDs;
+  isChecked?: boolean;
+  onToggle: () => void;
+};
+
+type CheckboxLabelMap = {
+  [key in CheckboxIDs]: {
+    cta: string;
+    accordion?: {
+      title: string;
+      content: string;
+    };
+  };
+};
+
+const CheckboxLabelKeys: CheckboxLabelMap = {
+  sendScreenshot: {
+    cta: I18n.t("contextualHelp.sendScreenshot.cta")
+  },
+  sendPersonalInfo: {
+    cta: I18n.t("contextualHelp.sendPersonalInfo.cta"),
+    accordion: {
+      title: I18n.t("contextualHelp.sendPersonalInfo.informativeTitle"),
+      content: I18n.t("contextualHelp.sendPersonalInfo.informativeDescription")
```

Comment:
If we use localized string in this way, they will never follow the current selected language.

May we convert this in function to execute inside component rendering?

Reply:
1. Ok for converting object map to function
2. The initial value was specified to be checked, but we changed our mind during the last review & planning meeting @Undermaken 
3. About the screenshot still being attached in the IB message, I'm not sure it should be handled in this PR

Reply:
about third point:

We cannot insert a fake checkbox, inside contextual help, so I guess this should be handled here or this PR should have another blocker and cannot be merged